core/common/src/main/java/alluxio/exception/ExceptionMessage.java

```diff
@@ -112,6 +113,7 @@
   FILE_ALREADY_EXISTS("{0} already exists"),
   FILE_CREATE_IS_DIRECTORY("{0} already exists. Directories cannot be overwritten with create"),
   HDFS_FILE_NOT_FOUND("File {0} with URI {1} is not found"),
+  PARENT_CREATION_FAILED_UFS("Unable to create parent directories for path {0}"),
```

Comment:
It doesn't seem like this error message is restricted to operations with the UFS? Not sure if you want to make it specific or general though.